qutebrowser/browser/hints.py

```diff
@@ -602,13 +604,15 @@ def _start_cb(self, elems):
                        star_args_optional=True, maxsplit=2)
     @cmdutils.argument('win_id', win_id=True)
     def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
-              *args, win_id, mode=None):
+              *args, win_id, mode=None, chronicle=False):
```

Comment:
Not sure if `--chronicle` is really a good name - while it's accurate, non-native English speakers (including me :wink:) probably won't know what it means at first sight. What about using `--history` instead?